hbase-server/src/main/java/org/apache/hadoop/hbase/master/MasterRpcServices.java

```diff
@@ -676,7 +676,16 @@ public BalanceResponse balance(RpcController controller,
       BalanceRequest request) throws ServiceException {
     try {
       return BalanceResponse.newBuilder().setBalancerRan(master.balance(
-        request.hasForce()? request.getForce(): false)).build();
+        request.hasForce() && request.getForce() ? LoadBalancer.RunMode.FORCE : LoadBalancer.RunMode.REQUEST)).build();
```

Comment:
This is getting a little complex. Could you pull getting the `RunMode` out of this line? (just do it in on its own line, please).